wordpress-mobile/WordPress-iOS
# Removes the `-en` from the Reader Interests slug key

Main Issue: https://github.com/wordpress-mobile/WordPress-iOS/issues/14396

Removes the `-en` from the Reader Interests slug key to adhere to the API update
Related: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/259

### Screenshots
<img width="30%" alt="Screen Shot 2020-07-14 at 9 18 50 AM" src="https://user-images.githubusercontent.com/793774/87450173-0ca24380-c5b3-11ea-85bd-fe97d0bd5f40.png">


### To test:
1. Launch the app
2. Enable the Reader Interests Phase 2 Feature Flag if it's not already enabled
3. Tap on the Reader tab
4. The Select Interests view should present itself
5. Verify the interests load

### PR submission checklist:
